tests/integration/clientv3/connectivity/dial_test.go

```diff
@@ -26,7 +26,7 @@ import (
 	"go.etcd.io/etcd/client/v3"
 	integration2 "go.etcd.io/etcd/tests/v3/framework/integration"
 	"go.etcd.io/etcd/tests/v3/framework/testutils"
-	clientv3test "go.etcd.io/etcd/tests/v3/integration/clientv3"
+	"go.etcd.io/etcd/tests/v3/integration/clientv3"
```

Comment:
It's updated by `goimports -w` automatically? Why did previous file add `clientv3test`, while this file remove `clientv3test`?

Reply:
goimports -w add this:
```go
	pb "go.etcd.io/etcd/api/v3/etcdserverpb"
	"go.etcd.io/etcd/client/pkg/v3/transport"
	integration2 "go.etcd.io/etcd/tests/v3/framework/integration"
	"go.etcd.io/etcd/tests/v3/framework/testutils"
	"go.etcd.io/etcd/tests/v3/integration/clientv3"
	clientv3test "go.etcd.io/etcd/tests/v3/integration/clientv3"
	"google.golang.org/grpc"
```

Mb I remove changes in tests?

Reply:
We want to keep 

```
clientv3test "go.etcd.io/etcd/tests/v3/integration/clientv3"
```

And change in this file goes in the other direction. So it's why we are surprised.

Reply:
Fixed.

I add `clientv3test "go.etcd.io/etcd/tests/v3/integration/clientv3"`myself and run goimports and all ok. Rerun goimports have no changes